app/Controllers/importExportController.php

```diff
@@ -255,11 +249,11 @@ private function ttrssXmlToJson(string $xml): string|false {
 				$item['categories'][] = 'user/-/state/com.google/broadcast';
 			}
 			if (!empty($item['label_cache'])) {
-				$labels_cache = json_decode($item['label_cache'], true);
+				$labels_cache = json_decode((string)$item['label_cache'], true);
 				if (is_array($labels_cache)) {
 					foreach ($labels_cache as $label_cache) {
 						if (!empty($label_cache[1])) {
-							$item['categories'][] = 'user/-/label/' . trim($label_cache[1]);
+							$item['categories'][] = 'user/-/label/' . trim((string)$label_cache[1]);
```

Comment:
It is quite incredible, but there are still many manual casts in this PR, so still not OK.
Those casts are not only unrelated to the PR, but several are also wrong, as one cannot cast `mixed` to `string`.
I will try to implement https://github.com/FreshRSS/FreshRSS/issues/6989 soonish, which will increase the typing in a better way.

Reply:
I'm going to delete all the casts I've added... 
For your information, trim() only accepts strings as the first parameter. 
The code above never checks that a string is present...
I delete them for this function. I can't guarantee that the code won't break if something else is entered as an argument.

Reply:
Sure, but:
1. It is unrelated to this PR;
2. It is not the correct fix.